### ISSUE TITLE
[Extensibility Enhancement] codeunit 5069 Word Template Interactions

### ISSUE THREAD

**Issue by akontos** [microsoft/ALAppExtensions#15862]
Hello. In an attempt to use Codeunit 5069 Word Template Interactions i have noticed that its Access is marked as Internal without any obvious Reason since you also have Events in the codeunit. It would be great if you could change the Access to be public

```
codeunit 5069 "Word Template Interactions"
{
     //Access = Internal;  <--- OLD CODE
....
}

```

Thank You

**Comment by dnpb** [comment]
seems a dupe of https://github.com/microsoft/ALAppExtensions/issues/15021

**Comment by mazhelez** [comment]
The bug is indeed a duplicate. The issue has been addressed and the fix is available in version 19.2.
